### DISCUSSION TITLE
Combining projects into one

### DISCUSSION THREAD

**Discussion by ivana67dj** [keiwando/toonsquid#92] [Q&A] [answered]
Hello! I just noticed that there is a scene feature! I was wondering whether it is possible to combine multiple projects into one using that feature

**Comment by Lioncub901** [comment]
But is that not why the scene menu is created? What do you mean and why do you need it? (I am not the creator just a user)

**Reply by ivana67dj** [reply]
As I said, I have already created a bunch of different parts to my project and I would like to combine them into one

**Reply by ivana67dj** [reply]
Just wondering because it seems like that might be the only way to copy and paste different elements and symbols across different parts of the short film

**Reply by Lioncub901** [reply]
Oh so you made a mistake and did not use the scene menu. Am I correct? If that is so, maybe that might be a good idea the only problem is that if your projects have different screen resolutions.

**Comment by keiwando** [comment]
If you want to share symbols between multiple parts of a short film then yes, you have to use a single project in which you can then create separate scenes. The only limitation is that all scenes have the same resolution and frame rate as the project itself (which generally shouldn't be an issue).
There is currently no way to combine multiple separate ToonSquid projects into one.

**Comment by keiwando** [comment, marked as answer]
You can now "merge" multiple project files into one with the 1.9 update by importing a project file (`.tsproj`) into another via the library.
This is mainly to make it possible to export selections of assets into separate reusable libraries but those file are also just regular `.tsproj` files, so you can also import any previously exported project file into another project.

https://toonsquid.com/handbook/export/project/#asset-library-export-import
